asyncpgsa/connection.py

```diff
@@ -15,32 +15,41 @@
 _dialect.paramstyle = 'named'
 
 
+def replace_keys(keys, querystring, params, inline=False):
+    new_query = querystring
```

Comment:
seeing as how `querystring` is never used again in this function we can probably just use that variable the whole time.